src/coreclr/src/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -360,6 +360,12 @@ internal static RuntimeAssembly InternalLoad(AssemblyName assemblyName, ref Stac
         // given name.  (Name should not include path.)
         public override FileStream? GetFile(string name)
         {
+            if (Location == "")
```

Comment:
Nit, Location declares it's not nullable, so I guess you can use `Location.Length  == 0` which avoids the call to Equals.